### ISSUE TITLE
[BERT] Ensure hyperparameters are the same

### ISSUE THREAD

**Issue by MarkusSagen** [Hugging-Face-Supporter/TFTokenizers#2]
Tokenization methods should use the same hyperparameters, s.a: 
- Truncation
- Padding
- Defining all the special characters (<UNK>, <CLS>, etc.)
- Vocab size
- Max sequence length

**Comment by MarkusSagen** [comment]
```
>>> from transformers import AutoTokenzier
>>>hf_tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")

>>> hf_tokenizer.init_kwargs
{'do_lower_case': True,
 'unk_token': '[UNK]',
 'sep_token': '[SEP]',
 'pad_token': '[PAD]',
 'cls_token': '[CLS]',
 'mask_token': '[MASK]',
 'tokenize_chinese_chars': True,
 'strip_accents': None,
 'model_max_length': 512,
 'special_tokens_map_file': None,
 'name_or_path': 'bert-base-uncased'}
>>> hf_tokenizer.model_input_names
['input_ids', 'token_type_ids', 'attention_mask']

>>> hf_tokenizer.pretrained_init_configuration
{'bert-base-uncased': {'do_lower_case': True},
 'bert-large-uncased': {'do_lower_case': True},
 'bert-base-cased': {'do_lower_case': False},
 'bert-large-cased': {'do_lower_case': False},
 'bert-base-multilingual-uncased': {'do_lower_case': True},
 'bert-base-multilingual-cased': {'do_lower_case': False},
 'bert-base-chinese': {'do_lower_case': False},
 'bert-base-german-cased': {'do_lower_case': False},
 'bert-large-uncased-whole-word-masking': {'do_lower_case': True},
 'bert-large-cased-whole-word-masking': {'do_lower_case': False},
 'bert-large-uncased-whole-word-masking-finetuned-squad': {'do_lower_case': True},
 'bert-large-cased-whole-word-masking-finetuned-squad': {'do_lower_case': False},
 'bert-base-cased-finetuned-mrpc': {'do_lower_case': False},
 'bert-base-german-dbmdz-cased': {'do_lower_case': False},
 'bert-base-german-dbmdz-uncased': {'do_lower_case': True},
 'TurkuNLP/bert-base-finnish-cased-v1': {'do_lower_case': False},
 'TurkuNLP/bert-base-finnish-uncased-v1': {'do_lower_case': True},
 'wietsedv/bert-base-dutch-cased': {'do_lower_case': False}}

>>> hf_tokenizer.SPECIAL_TOKENS_ATTRIBUTES
['bos_token', # beginning of string
 'eos_token', # end of string
 'unk_token', # OOV - unknown
 'sep_token', # Separation token, for when embedding two sentences
 'pad_token', # Padding batches to a fixed length. Carries no information
 'cls_token', # Tokens used for training and inference
 'mask_token',# Masked out tokens when pre-training the model
 'additional_special_tokens']

>>> hf_tokenizer.set_truncation_and_padding()
```
